build.sbt

```diff
@@ -21,9 +21,12 @@ inThisBuild(
         "scm:git:git@github.com:scalameta/metadoc.git"
       )
     ),
-    commands += Command.command("ci-release") { s =>
-      "very publishSigned" ::
-        s
+    publishTo := {
+      val nexus = "https://oss.sonatype.org/"
+      if (isSnapshot.value)
+        Some("snapshots" at nexus + "content/repositories/snapshots")
```

Comment:
Pro tip, `Opts.resolver.sonatypeStaging` and `Opts.resolver.sonatypeSnapshots` are a thing 😄

Reply:
Sweet